config/printer-creatbot-f430-v9.5.cfg

```diff
@@ -0,0 +1,245 @@
+# This file contains common pin mappings for the Creatbot F430
+# printer with V9.5 mainboard.
+
+# To use this config, the firmware should be compiled for the atmega2560.
+# When running "make menuconfig" you have to:
+#  Enable extra-low level options
+#  Micro-controller: Atmega AVR
+#  Processor model: atmega2560
+#  Processor speed: 16Mhz
+#  Communication: UART0
+#  Baud: 152000
+
+# See docs/Config_Reference.md for a description of parameters.
+
+#  NOTE: the following will no longer work
+#  Screen
+#  Udisk
+#  Power OFF switch (can use psu control in octoprint)
+
+## Printer ##
+
+[mcu]
+serial: /dev/ttyACM0
+baud: 115200
+
+[printer]
+kinematics: cartesian
+max_velocity: 200
+max_accel: 500
+max_z_velocity: 5
+max_z_accel: 100
+square_corner_velocity: 5.0
+
+## Steppers ##
+
+[stepper_x]
+step_pin: PH1
+dir_pin: !PH0
+enable_pin: !PL1
+microsteps: 16
+rotation_distance: 40.64
+endstop_pin: ^!PC0
+#endstop_pin: ^PE4 (additional endstop from original marlin)
+position_endstop: 0
+position_max: 400
+homing_speed: 100
+
+[stepper_y]
+step_pin: PL2
+dir_pin: !PF0
+enable_pin: !PF1
+microsteps: 16
+rotation_distance: 60.96
+gear_ratio: 50:20
+endstop_pin: ^!PC2 
+#endstop_pin: ^PJ0 (additional endstop from original marlin)
+position_endstop: 0
+position_max: 300
+homing_speed: 100
+homing_retract_dist: 5.0
+
+[stepper_z]
+step_pin: PF2
+dir_pin: PF3
+enable_pin: !PK0
+microsteps: 16
+rotation_distance: 4
+endstop_pin: probe:z_virtual_endstop
+position_max: 300
+##physical endstops if required##
+#endstop_pin: ^PC4 
+#endstop_pin: ^PD2 
+#position_endstop: 0.5
+
+## Extruders  ##
+ 
+[extruder]
+step_pin: PA0
+dir_pin: !PA1
+enable_pin: !PA2
+microsteps: 16
+rotation_distance: 22.5
+full_steps_per_rotation: 200
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PE4
+sensor_type: AD595
+sensor_pin: PK7
+#control: pid
+#pid_Kp: 12.9
+#pid_Ki: 0.76
+#pid_Kd: 55
+min_temp: 0
+max_temp: 265
+
+[gcode_macro T0]
+gcode:
+    SET_GCODE_OFFSET X=-36
+    ACTIVATE_EXTRUDER EXTRUDER=extruder
+
+[extruder1]
+step_pin: PA3
+dir_pin: !PA4
+enable_pin: !PA5
+microsteps: 16
+rotation_distance: 22.5
+full_steps_per_rotation: 200

```

Comment:
Not needed. Default

Reply:
which bit?

Reply:
Comment always apply to the line directly above

Reply:
removed